accounts/forms.py

```diff
@@ -146,13 +144,9 @@ class RegistrationForm(forms.Form):
     recaptcha_response = forms.CharField(widget=CaptchaWidget, required=False)
     username = UsernameField()
 
-    first_name = forms.CharField(help_text=_("Optional."), max_length=30, required=False)
-    last_name = forms.CharField(help_text=_("Optional."), max_length=30, required=False)
     email1 = forms.EmailField(label=_("Email"), help_text=_("We will send you a confirmation/activation email, so make "
```

Comment:
does it make sense to have the "1" postfix? same for password...

Reply:
I would prefer not to do that now.